app/models/manager_refresh/target.rb

```diff
@@ -18,7 +18,17 @@ def initialize(association:, manager_ref:, manager: nil, manager_id: nil, event_
       @association = association
       @manager_ref = manager_ref
       @event_id    = event_id
-      @options     = options
+      @options     = options.except(:payload, :payload_id)
+
+      payload = options[:payload]
+      @options[:payload_id] = BinaryBlob.create!(:binary => payload).id if payload
+
+      payload_id = options[:payload_id]
+      if payload_id
+        payload_blob = BinaryBlob.find(payload_id)
```

Comment:
Can we move this to a separate call?

May need to change change k8s `InventoryCollectorMixin#parse_target_from_notice`
and TargetCollection#add_target

plus there are many uses of `load`